marc.py

```diff
@@ -453,6 +458,22 @@ def add_ebooks_subject(cls, record):
                 ]))
 
 
+class MARCExporterFacets(BaseFacets):
+    """A faceting object used to configure the search engine so that
+    it only works updated since a certain time.
+    """
+
+    def __init__(self, start_time):
+        self.start_time = start_time
+
+    def modify_search_filter(self, filter):
+        filter.order = self.SORT_ORDER_TO_ELASTICSEARCH_FIELD_NAME[
+            self.ORDER_LAST_UPDATE
+        ]
```

Comment:
The MARC file doesn't actually need to be sorted, only filtered. Does sorting make it easier to break the query into pages?

Reply:
The results need to be sorted because we're paginating with SortKeyPagination, but they don't need to be sorted by last update -- work ID would also be okay. But I don't think there's any harm in sorting them by last update, and it will ensure the inclusion of any updates that happen while we're building the MARC records.

Reply:
And why are we paginating with SortKeyPagination?

Reply:
Elasticsearch won't go beyond 100k records with normal pagination.